fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -145,6 +157,49 @@ def load_data(self, X, y, **kwargs):
         """Load the specified data into the object."""
         super().load_data(X, y, event=_ALL, **kwargs)
 
+class EqualOpportunity(ConditionalSelectionRate):
+    r"""Implementation of Equalized Odds as a moment.
```

Comment:
Documentation needs update

Reply:
unless this is meant to be Equalized Odds. The class name threw me off

Reply:
Sorry about that. I think all this should be resolved now!